lib/exabgp/reactor/loop.py

```diff
@@ -440,9 +440,8 @@ def restart (self):
 		self.processes.start()
 
 	def unschedule (self, peer):
-		key = peer.neighbor.name()
-		if key in self.peers:
-			del self.peers[key]
+		if peer in self.peers:
```

Comment:
This was done because with these changes neighbor.name() returns a different value for local_ip after we actually have connected and found our real ip address. Without this change we peers were not being unscheduled and this was causing problems.